apps/site/src/lib/blocks.ts

```diff
@@ -256,7 +262,15 @@ export const retrieveBlockFileContent = async ({
           { encoding: "utf8" },
         ),
       )
-    : await fetch(metadataSchemaUrl).then((response) => response.json());
+    : await fetch(metadataSchemaUrl)
+        .then((response) => response.json())
+        .catch((err) => {
+          // eslint-disable-next-line no-console -- intentional log to flag problem without tanking site
+          console.error(
+            `Could not fetch and parse schema at ${metadataSchemaUrl} for block ${pathWithNamespace}: ${err}`,
+          );
+          return { title: "Unparseable schema" };
+        });
```

Comment:
Drive-by: I had tests failing first time round and it was difficult to debug because there was a `500` error and the app crashing when a schema was unreachable. This handles the error a little better.

Reply:
Optional: It'd be great to avoid a mix of `async` / `await` syntax with `then` / `catch`.

Reply:
Sure – https://github.com/blockprotocol/blockprotocol/pull/923/commits/111d3bebf83071ffdab96ccfbcf9814c049d9760